### ISSUE TITLE
bug: cannot update ip_access list

### ISSUE THREAD

**Issue by tstraley** [ClickHouse/terraform-provider-clickhouse#29]
I have a `clickhouse_service` with 2 CIDRs in the `ip_access` list. When I attempted to remove one from the list, I get the following results:

plan (looks ok):
```
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.clickhouse.clickhouse_service.this will be updated in-place
  ~ resource "clickhouse_service" "this" {
      ~ iam_role             = "arn:aws:iam::<some_acct>:role/<some_IAM_Role>-Role" -> (known after apply)
        id                   = "caf76259-bcbb-4c20-93cf-dbdbd1a27c97"
      ~ ip_access            = [
          - {
              - description = "" -> null
              - source      = "<redacted>" -> null
            },
            # (1 unchanged element hidden)
        ]
      ~ last_updated         = "Sunday, 10-Dec-23 19:13:39 CST" -> (known after apply)
        name                 = "my-service-name"
        # (9 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

apply (fails):
```
Acquiring state lock. This may take a few moments...
module.clickhouse.clickhouse_service.this: Modifying... [id=caf76259-bcbb-4c20-93cf-dbdbd1a27c97]
╷
│ Error: Error Updating ClickHouse Service
│ 
│   with module.clickhouse.clickhouse_service.this,
│   on ../libs/clickhouse/main.tf line 8, in resource "clickhouse_service" "this":
│    8: resource "clickhouse_service" "this" {
│ 
│ Could not update service, unexpected error: status: 400, body:
│ {"requestId":"5cb84d86-2f04-429c-862a-9f5ae5741c1c","error":"BAD_REQUEST:
│ request body.ipAccessList.add[0]: [object]","status":400}
╵
Releasing state lock. This may take a few moments...
```

Using v0.0.4

**Comment by tstraley** [comment]
Looks like same error found in https://github.com/ClickHouse/terraform-provider-clickhouse/issues/11 but different cause and affecting latest version.

**Comment by tstraley** [comment]
As a workaround (for anyone else stuck on this) I was able to remove the IP access via the Clickhouse Cloud UI, and then a subsequent plan and apply worked without issue.
Obviously, this defeats the purpose of using terraform to define this and manage it as code, but gets me unblocked.

**Comment by Kinzeng** [comment]
this should be addressed in #34

**Comment by guykoh** [comment]
@Kinzeng is this fixed now? Can we close this issue?
